src/BuildEvents.h

```diff
@@ -11,7 +11,7 @@
 
 #ifdef _MSC_VER
 #define ftello64 _ftelli64
-#elif defined(__APPLE__) || defined(__FreeBSD__)
+#elif defined(__APPLE__) || defined(__FreeBSD__) || (defined(__linux__) && !defined(__GLIBC__))
```

Comment:
It might be advisable to move this check somewhere else (like a dedicated header), as right now `ftello64()` is used not only in `BuildEvents.cpp`, but also in `main.cpp`. At the latter place, compilation only succeeds since `BuildEvents.h` gets included (which is a kind of a hidden dependency).

Reply:
Yeah it is a bit of a mess, I'll try to remember to do something about it.